Update docs to reflect device_class changes for vizio component

raman325 opened the pull request:
**Description:** Made a breaking change to the available `device_class` options to use the device classes defined in `media_player`. This has the benefit of making all `vizio` devices compatible with Alexa and Google Assistant.


**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#30802

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

Commit:
update docs to reflect change from platform level to config level entry

raman325 committed:
update docs to reflect device_class changes and add new options

raman325 committed:
fix int type for config parameters

frenck reviewed (dismissed):
✅ Approved. Can be merged as soon as the parent PR gets merged.

raman325 committed:
remove timeout option per corresponding home-assistant PR review

raman325 commented:
I removed the new option I had originally added to the corresponding PR (home-assistant/home-assistant#30802) based on review feedback so I updated this PR accordingly
